specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3811,6 +4562,164 @@
+              "type": "string",
+              "readOnly": true
+            },
+            "connectionString": {
+              "description": "The connection string to use to connect to the database.",
+              "type": "string",
+              "readOnly": true
+            },
+            "region": {
+              "description": "The region of the database resource.",
+              "type": "string",
+              "readOnly": true
+            },
+            "configurationFiles": {
+              "description": "A list of configuration files associated with this database connection.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/StaticSiteDatabaseConnectionConfigurationFileOverview"
+              },
+              "x-ms-identifiers": [],
+              "readOnly": true
+            },
+            "name": {
+              "description": "If present, the name of this database connection resource.",
+              "type": "string",
+              "readOnly": true
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DatabaseConnectionResponseCollection": {
+      "description": "Collection of static site database connections.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatabaseConnectionResponse"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DatabaseConnectionPatchRequest": {
+      "description": "Static Site Database Connection Request Properties ARM resource when patching",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "DatabaseConnectionPatchRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "resourceId": {
+              "description": "The resource id of the database.",
+              "type": "string"
+            },
+            "connectionIdentity": {
+              "description": "If present, the identity is used in conjunction with ConnectionString to connect to the database. Use of the system-assigned managed identity is indicated with the string 'SystemAssigned', while use of a user-assigned managed identity is indicated with the resource id of the managed identity resource.",
+              "type": "string"
+            },
+            "connectionString": {
+              "description": "The connection string to use to connect to the database.",
+              "type": "string"
+            },
+            "region": {
+              "description": "The region of the database resource.",
+              "type": "string"
+            }
```

Comment:
I am a bit confused .. you have defined these as readonly in the PUT but you callow the user to update them on PATCH ?